src/language/typescript.ts

```diff
@@ -143,13 +143,14 @@ const getRelativeRefPath = (cwd: string, refBlockName: string, refFileName: stri
 	`${getRelativeRoot(cwd)}/${refBlockName}/${refFileName}`;
 const getRelativeOutRefPath = (cwd: string, blockName: string, outFileName: string, refFileName: string): string =>
 	`${getRelativeRoot(cwd)}/../${outFileName}/${blockName}/${refFileName}`;
-const getRelativeClientPath = (cwd: string): string => `${getRelativeRoot(cwd)}/${CLIENT_DIRECTORY}/${CLIENT_FILENAME}`;
-const getRelativeUtilsPath = (cwd: string): string => `${getRelativeRoot(cwd)}/${UTILS_DIRECTORY}/${UTILS_FILENAME}`;
+const getRelativeClientPath = (cwd: string): string =>
+	`../${getRelativeRoot(cwd)}/${CLIENT_DIRECTORY}/${CLIENT_FILENAME}`;
```

Comment:
Actually I'm against hardcoding the paths (`../`), the same applies to `getRelativeOutRefPath`.